package.json

```diff
@@ -45,7 +45,7 @@
     "dotenv": "^8.0.0",
     "express-extractheaders": "^3.0.0",
     "iltorb": "^2.4.3",
-    "polyfill-library": "^3.40.0",
+    "polyfill-library": "^3.42.0",
```

Comment:
what happened to 3.41 !

Reply:
It exists, we just didn't release a version of polyfill.io with it.